Olympus.Stratis/initPlayerLocal.sqf

```diff
@@ -0,0 +1,27 @@
+/*
+ * Author: Theseus Incorporated
+ *
+ * initPlayerLocal.sqf
+ * Initialization of scripts local to player
+ * Executed locally when player joins mission (includes both mission start and JIP)
+ * Executed after initServer and before initPlayerServer and init.
+ *
+ * Arguments: [player:Object, didJIP:Boolean]
+ *
+ */
+
+#include "script_component.hpp"
+
+[] call FUNC(healCommand);
```

Comment:
Might as well name it `addChatCommands`, they are usually super short. Also no need for empty array parameter if it doesn't need any.